src/widget/wtrackmenu.cpp

```diff
@@ -1795,7 +1796,8 @@ void WTrackMenu::slotRemoveFromDisk() {
     }
 
     const QList<QString> tracksToKeep(trackOperator.getTracksToKeep());
-    if (!tracksToKeep.isEmpty()) {
+    if (tracksToKeep.isEmpty()) {
+        // All selected tracks could be processed. Finish!
```

Comment:
Please remove the redundant extra scope following this early return while you are at it. We now know that this code didn't change except its indentation ;)

Reply:
ahhm, sure :)